src/js/content_scripts/input-mapper.js

```diff
@@ -28,6 +28,14 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 funcName: "gamepad.inputMapper.restoreFocus",
                 args: ["{that}.options.windowObject", "{that}.tabindexSortFilter"]
             },
+            goToPreviousTab: {
```

Comment:
Since you already have invokers, if you were to standardise to a single function, you could just indicate the direction here.